lib/active_merchant/billing/gateways/trans_first_transaction_express.rb

```diff
@@ -438,14 +552,14 @@ def refund_type(action)
       end
 
       # -- request methods ---------------------------------------------------
-      def build_xml_transaction_request
-        build_xml_request('SendTranRequest') do |doc|
+      def build_xml_transaction_request(merchant_product_type = nil)
+        build_xml_request('SendTranRequest', merchant_product_type) do |doc|
```

Comment:
all 3 of these xml changes: https://github.com/rednovalabs/active_merchant/pull/9/commits/4b03ce2e7d83e3d5b48344616041129e3a6c9f58